lumen/app/Console/Commands/ImportEvents.php

```diff
@@ -21,11 +21,16 @@ public function __construct(EventDataProvider $eventDataProvider) {
 
     public function handle() {
         $location = $this->argument('location');
-        
-        $events = $this->eventDataProvider->getByLocation($location);
-
-        foreach ($events as $event) {
-            dispatch(new \App\Jobs\ImportEvent($event));
+        $page = 1;
+        $has_more_items = true;
+        while ($has_more_items) {
```

Comment:
it's fine, we could improve this using a `do while` or even return an iterator from the data provider to handle the iteration over the pagination.